src/app/pipes/timedisplay.ts

```diff
@@ -0,0 +1,53 @@
+import { PipeTransform, Pipe } from '@angular/core';
+
+
+@Pipe({name: 'timeDisplay'})
+export class TimeDisplayPipe implements PipeTransform {
+    transform(timeInSeconds: number, shortened = false): string {
```

Comment:
Add a check to make sure that `timeInSeconds` is not null. If it is, it should just return an empty string.